install-dependencies.sh

```diff
@@ -52,6 +52,10 @@ debian_base_packages=(
     slapd
     ldap-utils
     libcpp-jwt-dev
+    elfutils
+    curl
+    jq
+    git-lfs
```

Comment:
what is this needed for?

Reply:
Please explain in the commit message every dependency. If "git-lfs" really needed? How come it's needed only for Debian and not for Fedora (below)?

Regarding curl, I see we already had it listed for Fedora but not Debian. I wonder if it was an oversight, or curl comes pre-installed on Ubuntu (I didn't think it was, but maybe it is?)

Reply:
Ok, I didn't understand you were refering to the existing scripts/open-coredump.sh. I see it indeed uses jq, so I guess that's wher this dependency comes from. I still don't know why specifically git-lfs (not just git) is required. Do we need git-lfs to clone a git repo? If you just want to extract a specific file, maybe we don't need an entire clone (and in particular don't need git-lfs)?

Reply:
Yes, I was specifically asking about just `git-lfs`, the others LGTM.
I don't see (based on a quick glance) `git-lfs` in the Fedora package list, so I'm curious why it is needed for ubuntu.

Reply:
when I ran the script the first time, something was complaining that it needs `git-lfs` to run, I didn't investigate what part of the script needs it though. It was already listed in the Fedora packages, it's just above near `git`, not visible in this diff, so I added it for debian as well.

Reply:
ok, fair enough